docs/platform.rst

```diff
@@ -82,6 +84,35 @@ The typical things an engine needs to handle are:
   same base class, apps can call methods on the engines to for example create UIs. It is up to each
   engine to implement these methods so that they work nicely inside the host application.
 
+- A startup interface for related DCC applications.
+
```

Comment:
Perhaps an example use case or two highlighting the different scenarios might be helpful as well. You could use these scenarios to provide additional examples in the bullet points below where you talk about the various requirements and features.